layouts/partials/list.html

```diff
@@ -14,13 +14,13 @@
     <article class="card mb-3">
       <div class="row g-0">
         <div class="col-md-4">
-          {{ partial "helpers/image" (dict "filename" (printf "%s%s" .Params.image "?width=400px") "alt" .Title "class" "img-fluid rounded-start" "resources" .Resources) }}
+          {{ partial "helpers/image" (dict "filename" (printf "%s%s" .Params.image "?width=400px") "alt" "" "class" "img-fluid rounded-start" "resources" .Resources) }}
```

Comment:
Wound't be better for accessibility if we had the alternate text so screen readers could read it?

Reply:
@elyezer the reasoning here is that currently the alt text is just the post title, which is already available for screenreaders to read it

By replacing it with an empty alt, the image is considered decorative and is skipped, so the post title is not read twice

Reply:
This is a great suggestion, thank you!